runners/sbt/src/main/scala/stryker4s/sbt/Stryker4sPlugin.scala

```diff
@@ -15,17 +16,29 @@ object Stryker4sPlugin extends AutoPlugin {
   override def trigger = allRequirements
 
   object autoImport {
+    val Stryker = config("stryker") extend Test
```

Comment:
Are you sure we need a new Configuration for this? `logLevel in stryker` (the task) also compiles

Reply:
You are very correct, we don't need this.